src/server/graphql/resolvers/index.js

```diff
@@ -155,7 +155,18 @@ export async function resolveProjectUserSummaries(projectSummary, args, {rootVal
 }
 
 export function resolveStartOfWeek(parent) {
-  return parent.weekStartedAt || moment(parent.startTimestamp).startOf('isoweek').toDate()
+  if (parent.weekStartedAt) {
+    return parent.weekStartedAt
+  }
+  if (!parent.startTimestamp) {
+    return moment().startOf('isoweek').toDate()
```

Comment:
If the parent doesn't have a `startTimestamp` value, let's use the `createdAt` value.